File: 001-multiples-of-3-and-5/main.py
if __name__ == "__main__":
    # Get threes and fives
    threes = range(3, 1000, 3)
    fives = range(5, 1000, 5)
    # To eliminate duplicates, create a set of one and union the other with it
    unique_vals = set(threes).union(set(fives))
    print(sum(unique_vals))

File: 001-multiples-of-3-and-5/main.R
# First get the multiples of 3 below 1000
threes <- seq(3, 999, by = 3)
# Then the multiples of 5
fives <- seq(5, 995, by = 5)
# Take only the unique values to eliminate those that are multiples of 15
unique_vals <- unique(c(threes, fives))
sum(unique_vals)
